static/custom.css

```
@@ -1,7 +1,17 @@
 body { background-color: var(--bs-dark) !important;
        color: var(--bs-light);
 }
-
 .row{
   margin-bottom: 5%;
+}
+
+.footer{
+  display: table;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
```

```suggestion
  width: 100%;
  font-size: small;
```
I think adding `font-size: small;` here makes it fit/look a bit better on the screen :slightly_smiling_face: